tool/teleport/common/sftp.go

```diff
@@ -174,6 +261,9 @@ func (s *sftpHandler) Filecmd(req *sftp.Request) (retErr error) {
 	if req.Filepath == "" {
 		return os.ErrInvalid
 	}
+	if err := s.ensureReqIsAllowed(req); err != nil {
+		return err
```

Comment:
Should we be `trace.Wrap`ping these errors?

Reply:
These get returned to `github.com/pkg/sftp` code so the trace wouldn't be very useful, I can add them if you think it's still worth doing though.

Reply:
Got it. Sounds fine.